homeassistant/components/zha/__init__.py

```diff
@@ -184,7 +184,16 @@ async def async_migrate_entry(
             data[CONF_DEVICE][CONF_BAUDRATE] = baudrate
 
         config_entry.version = 2
-        hass.config_entries.async_update_entry(config_entry, data=data)
```

Comment:
[The documentation](https://developers.home-assistant.io/docs/config_entries_config_flow_handler/#config-entry-migration) says to modify `config_entry.data` directly instead of calling `hass.config_entries.async_update_entry`.

Is this still the correct way to do it?

Reply:
I think you are correct.

Reply:
No, the docs are outdated. Please use `hass.config_entries.async_update_entry` to update the entry data.